pornography-hosts

```diff
@@ -15278,3 +15278,6 @@
 0.0.0.0 gotxx.com
 0.0.0.0 mydesi.net
 0.0.0.0 nuefliks.club
+0.0.0.0 desixflix.com
+0.0.0.0 masturbate.com
+0.0.0.0 eporner.net
```

Comment:
Redirects to `masturbate.com`: 

`eporner.net`: https://mypdns.org/my-privacy-dns/porn-records/-/issues/477